DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,7 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
+                ->scalarNode('default_formatter')->end() // TODO: make it required when the major version is changed
```

Comment:
Because the list of supported formatters are also defined in the config. Supported formatters are the keys defined in the formatters section.